Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,24 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    expand = File.expand_path("~/Applications")
+    File.exist?("#{expand}/Firefox.app")
```

Comment:
@mikemcquaid I've tried something, but it needs someone on Mavericks or below to double check it functions as expected when passing ` --without-xulrunner `.